src/plugins/vctlGit.nim

```diff
@@ -278,15 +279,18 @@ template parseTime(line: string): int =
 template formatCommitObjectTime(line: string): string =
   fromUnix(parseTime(line)).format(gitTimeFmt) & " " & line.split()[^1]
 
-proc readPackedCommit(path: string, offset: uint64): string =
+proc readPackedObject(path: string, offset: uint64): string =
+  ## read packaged git object
+  ## supports reading git commit and tag objects
   withFileStream(path, mode = fmRead, strict = true):
     stream.setPosition(int(offset))
     let initialReadSize = 0x1000
     var
-      data = stream.readStr(initialReadSize)
-      byte = uint8(data[0])
-    if ((byte shr 4) and 7) != gitObjCommit:
-      raise(newException(CatchableError, "invalid commit object"))
+      data    = stream.readStr(initialReadSize)
+      byte    = uint8(data[0])
+      objType = ((byte shr 4) and 7)
+    if not (objType == gitObjCommit or objType == gitObjTag):
```

Comment:
`notin` didnt seem to work with `uint8` and `int` comparison although either way this is faster as it can do direct boolean logic vs scanning a list

cc @drraid we actually spoke about it with him while pairing on a fix as he wrote all the original logic for the packed files